src/Mauro/Greeter.test.ts

```diff
@@ -1,42 +1,15 @@
 import { Greeter } from "./Greeter";
 
 describe("Greeter.greet", () => {
-  it("should receive a name return a message Hello 'name'", () => {
+  it("should return 'Good morning <name>' when the time is 06:00-12:00", () => {
     // arrange
-    const greeter = new Greeter();
-
-    // act
-    const result = greeter.greet("Mauro");
-
-    // assert
-    expect(result).toBe("Hello Mauro");
-  });
-
-  it("should trim received name", () => {
-    // arrange
-    const greeter = new Greeter();
-
-    // act
-    const result = greeter.greet("  Mauro  ");
-
-    // assert
-    expect(result).toBe("Hello Mauro");
-
-    // act
-    const result2 = greeter.greet("Mauro  ");
-
-    // assert
-    expect(result2).toBe("Hello Mauro");
-  });
```

Comment:
Did you delete this test? I think that it would be useful to keep it in case of having a reggression.